src/test/system_tests/ceph_s3_tests/test_ceph_s3.js

```diff
@@ -100,29 +99,35 @@ async function test_worker() {
     }
 }
 
-async function run_single_test(test) {
-    let ceph_args = `S3TEST_CONF=${CEPH_TEST.test_dir}${CEPH_TEST.ceph_config}`;
-    if (S3_CEPH_TEST_SIGV4.includes(test)) {
+async function run_single_test(test_name) {
+    let ceph_args = `S3TEST_CONF=${process.cwd()}/${CEPH_TEST.test_dir}${CEPH_TEST.ceph_config}`;
+    if (S3_CEPH_TEST_SIGV4.includes(test_name)) {
         ceph_args += ` S3_USE_SIGV4=true`;
     }
-    let base_cmd = `${ceph_args} ./${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}virtualenv/bin/nosetests`;
-    if (!S3_CEPH_TEST_OUT_OF_SCOPE_REGEXP.test(test)) {
+    let base_cmd = `${ceph_args} tox ${TOX_ARGS}`;
+    if (!S3_CEPH_TEST_OUT_OF_SCOPE_REGEXP.test(test_name)) {
         try {
-            const test_name = test.replace(S3_CEPH_TEST_STEMS_REGEXP, pref => `${pref.slice(0, -1)}:`); //Match against the common test path
             if (test_name.includes('boto')) {
-                base_cmd = `${ceph_args} ./${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}virtualenv/bin/nosetests -v -s -A 'not fails_on_rgw'`;
+                base_cmd = `${ceph_args} tox ${TOX_ARGS} -- -m 'not fails_on_aws'`;
             }
-            const res = await os_utils.exec(`${base_cmd} ${test_name}`, { ignore_rc: false, return_stdout: true });
-            if (res.indexOf('SKIP') >= 0) {
-                console.warn('Test skipped:', test);
-                testing_status.skip.push(test);
+            const res = await os_utils.exec(`${base_cmd} ${process.cwd()}/${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}${test_name}`, { ignore_rc: false, return_stdout: true });
```

Comment:
My intention was to make this line more readable.
If you think it can help, change it, if not just revert to the variable name in the previous comment.

```suggestion
            const full_test_name = `${base_cmd} ${process.cwd()}/${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}${test_name}`;
            const res = await os_utils.exec(full_test_name, { ignore_rc: false, return_stdout: true });
```

Reply:
sure, although this is not the full test name but the entire test command, maybe I can call the variable something like full_test_command